app/views/contexts/section_summary_context.py

```diff
@@ -181,6 +181,7 @@ def _custom_summary_elements(
                     schema=self._schema,
                     location=self.current_location,
                     language=self._language,
+                    return_to="section-summary",
```

Comment:
How come we default this here, we might now always have a section summary. We should have no summary, a section summary or a list summary.

Reply:
`ListCollectorBlock` previously would default to section-summary in `list_summary_element` and now that its a configurable param I need to add it, I think from `section_summary_context.py` this is valid to pass in?

Reply:
Ya, it was more around hard coding it. It might be an existing issue.

Reply:
Still outstanding?.

Reply:
Would it not be "section-summary" for `section_summary_context` ? 
I've not got an instance attribute I can set for this and the function doesn't accept it as a parameter, so I've added the return to here as ListCollectorBlock doesn't default it to "section-summary" anymore

I could change `SectionSummaryContext` to accept a return to parameter though I think it would still be hardcoded somewhere
I might have missed what you had in mind?

Reply:
🤦 My bad, was looking at the wrong file. Yes this is correct since it is the section summary context.